omod/src/main/webapp/resources/css/ugandaemr-poc-styles.css

```diff
@@ -228,3 +228,10 @@ span.time-box-horizontal select {
     display: inline-block;
     min-width: 20% !important;
 }
+
+/*
+* This class provides a gray background on textfields which are inactive/disabled
+*/
+span#bg-color-gray input{
```

Comment:
@semujju use a class html-form-entry-disabled-field so that its very clear on naming and usage

Reply:
Done